assets/src/edit-story/components/form/media.js

```diff
@@ -203,7 +204,7 @@ function MediaInput({
       {...(canReset && resettableProps)}
     >
       {value && !isMultiple ? (
-        <Img src={value} circle={circle} />
+        <Img src={value} circle={circle} alt={alt} />
       ) : (
         <DefaultImage size={size} />
```

Comment:
How is this announced by screen readers? Does it also needs some alt / aria-label / title?

Reply:
I honestly added this to make the accessibility e2e test pass. I think we should look at this in another PR / issue.

Reply:
Please create an issue

Reply:
And perhaps just set `alt=""` in the meantime on the `Img`

Reply:
I think the default we have now it pretty solid. We can change it later.

Reply:
I have created a breakout issue here https://github.com/google/web-stories-wp/issues/5452